documentation/docs/mock-apps/services/manage-vc-status.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 28
+title: Manage Verifiable Credential Status
+---
+
+import Disclaimer from '../../\_disclaimer.mdx';
+
+<Disclaimer />
+
+## Description
+
+The Manage Verifiable Credential (VC) Status service is responsible for managing the status of a VC is an important feature of the Verifiable Credential ecosystem. This document explains how to activate and revoke VCs using the VCKit API.
+
+## API Endpoint for Managing VC Status
+
+To manage the status of a Verifiable Credential (activate or revoke), you will use the following API endpoint in the VCKit API:
+
+Set Bitstring Status API Endpoint: `POST http://localhost:3332/agent/setBitstringStatus`
+
+## Diagram
+
+The sequence diagram below illustrates the flow of managing the status of a VC using the `Set Bitstring Status API`:
+
+```mermaid
+
+sequenceDiagram
+participant U as User
+participant V as VCKit
+
+U->>V: Call setBitstringStatus(payload)
+V->>V: Validate payload
+V->>V: Update VC status
+V-->>U: Return status update
+
+```
+
+## Example Request Payload
+
+The request payload for updating the status of a VC should look like this:
+
+```json
+{
+  "statusListCredential": "http://localhost:3332/credentials/status/bitstring-status-list/26",
+  "statusListVCIssuer": "did:web::example.com",
+  "statusPurpose": "revocation",
+  "index": 0,
+  "status": false
+}
+```
+
+## Definitions
+
+| Property             | Required | Description                                                                                                   | Type    |
+| -------------------- | -------- | ------------------------------------------------------------------------------------------------------------- | ------- |
+| statusListCredential | Yes      | The URL of the status list associated with the VC. This comes from the `credentialStatus` property in the VC. | String  |
+| statusListVCIssuer   | Yes      | The DID of the issuer of the status list, typically your DID (e.g., did:web:...)                              | String  |
+| statusPurpose        | Yes      | The purpose for updating the status, which can be `revocation` or other purposes like suspension.             | String  |
+| index                | Yes      | The position of the status being updated in the status list, found in the `credentialStatus` of the VC.       | Integer |
+| status               | Yes      | The status of the credential. If `false`, the credential is active. If `true`, the credential is revoked.     | Boolean |
+
+## Retrieving Credential Status Information
+
+To manage the status of a VC, you will need to retrieve the status information from the VC. Here's how you can do that:
+
+1.	Verify the VC: In the mock app, go to the Verify page and scan the barcode associated with the VC.
```

Comment:
Hi, I have a little confused with Verify page and scan the barcode. As I understand, Verify page is "/verify" and to scan a barcode, we need to visit "/scanning". Is it correct?

Reply:
Yes, that's correct. I'll update it.